From 63e0780380d7c7254bdae0ba1d0009164b4ef5f1 Mon Sep 17 00:00:00 2001
From: Yuantian Tang <andy.tang@nxp.com>
Date: Sun, 23 Feb 2025 22:47:13 -0800
Subject: [PATCH] Change Recipe Target Sysroot patch

Signed-off-by: Andy Tang <andy.tang@nxp.com>
---
 voiceAction/Makefile | 14 ++++++++++----
 1 file changed, 10 insertions(+), 4 deletions(-)

diff --git a/voiceAction/Makefile b/voiceAction/Makefile
index d246a2a..3a30673 100644
--- a/voiceAction/Makefile
+++ b/voiceAction/Makefile
@@ -3,10 +3,13 @@ BLDDIR=build
 TEST_FILES= \
 	BTPVA_action_executor.c
 
-IFLAGS=-I/${OECORE_TARGET_SYSROOT}/usr/include/glib-2.0 \
-       -I/${OECORE_TARGET_SYSROOT}/usr/lib/glib-2.0/include -lglib-2.0 \
-       -I/${OECORE_TARGET_SYSROOT}/usr/include/dbus-1.0/ \
-       -I/${OECORE_TARGET_SYSROOT}/usr/lib/dbus-1.0/include/
+IFLAGS=-I/${RFSDIR}/usr/include/glib-2.0 \
+       -I/${RFSDIR}/usr/lib/glib-2.0/include -lglib-2.0 \
+       -I/${RFSDIR}/usr/include/dbus-1.0/ \
+       -I/${RFSDIR}/usr/lib/dbus-1.0/include/ \
+       -I/${RFSDIR}/usr/lib/aarch64-linux-gnu/ \
+       -I/${RFSDIR}/usr/lib/aarch64-linux-gnu/glib-2.0/include/ \
+       -I/${RFSDIR}/usr/lib/aarch64-linux-gnu/dbus-1.0/include/ \
 
 CFLAGS=-std=gnu99 \
 	 -Wno-unused-variable \
@@ -50,6 +53,9 @@ $(BLDDIR)/.mkdir :
 $(EXE) : $(EXE_OBJ_FILES)
 	$(CC) $(CFLAGS)  $^  \
 	 -lm  \
+	 -ldbus-1 \
+	 -lglib-2.0 \
+	 -lsystemd \
 	  -o $@
 
 $(BLDDIR)/%.o : %.c $(BLDDIR)/compiler_flags
-- 
2.34.1

