From f1e310312de8a45cadc76e2b818276225698908c Mon Sep 17 00:00:00 2001
From: Yuantian Tang <andy.tang@nxp.com>
Date: Tue, 5 Aug 2025 19:52:19 -0700
Subject: [PATCH] Remove regor since it was not used

Signed-off-by: Andy Tang <andy.tang@nxp.com>
---
 ethosu/vela/vela.py | 1 -
 1 file changed, 1 deletion(-)

diff --git a/ethosu/vela/vela.py b/ethosu/vela/vela.py
index ac4fb62..1157d6e 100755
--- a/ethosu/vela/vela.py
+++ b/ethosu/vela/vela.py
@@ -56,7 +56,6 @@ from .tflite_model_semantic import TFLiteSemantic
 from .tflite_supported_operators import TFLiteSupportedOperators
 from .tosa_model_semantic import TosaSemantic
 from .tosa_supported_operators import TosaSupportedOperators
-from ethosu import regor
 
 TFLITE_MAGIC = 0x334C4654
 TOSA_MAGIC = 0x41534F54
-- 
2.34.1

